src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1536,6 +1536,24 @@ public function isMethodCacheable()
     }
 
     /**
+     * Returns the original protocol version.
```

Comment:
This description should be improved to explain very briefly what does "the original protocol" mean. Thanks!